### PR TITLE
Add troubleshooting section #48

### PR DESCRIPTION
**Pull request by LeoMouyna**
- Add troubleshooting section on readme documentation.
- Solve missing cpp libs problem.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,13 +11,21 @@ He is able to kill Impostors. If he shoots a Crewmate, he will lose his life ins
...
 
-<h2> Installation </h2>
+<h2 id="installation"> Installation </h2>
 <ul>
 <li>Download the Mod for your specific game version. You are not able to launch the game if the versions do not match.</li>
 <li>Make a copy of your game’s root directory (Steam/steamapps/common/Among Us) and rename it to whatever you want (Steam/steamapps/common/Among Us Sheriff Mod) </li>
 <li>Extract the content of Among Us Sheriff Mod.zip into the copied folder you created</li>
 <li>Open your modded folder and open the Game via Among Us.exe</li>
 </ul>
+<p>To check if installation succeed<p>
```
Comment (TomDufall) [inline README.md:21]:
```suggestion
<p>Verifying installation success<p>
```

File: README.md [hunk 2]
```diff
@@ -11,13 +11,21 @@ He is able to kill Impostors. If he shoots a Crewmate, he will lose his life ins
...
+<ul>
+  <li>Launch the Game via Among Us.exe.
+  <li>On top, left corner, below among us version you should see <em>loaded Sheriff Mod vx.y by Woodi </em>
```
Comment (TomDufall) [inline README.md:24]:
```suggestion
  <li>In the top-left corner, below Among Us version, you should see <em>loaded Sheriff Mod vx.y by Woodi </em>
```

File: README.md [hunk 3]
```diff
@@ -70,6 +78,22 @@ Yes. Every player in the game lobby has to install it.</p>
 I would appreciate any donations. This will help me to develop more mods.
 
 For exclusive modding requests contact me: <a href="mailto:Woodi-dev@gmx.de">Woodi-dev@gmx.de</a>
+
+<h2 id="troubleshooting">Troubleshooting</h2>
+
+<p><b>I can't see <em>loaded</em> message on my game screen</b></br>
+<ol>
+  <li>Make sure you followed all <a href="#installation">instalation steps</a>, especially launching game via Among Us.exe file</li>
+  <li>You migh missing some cpp libs, please install
```
Comment (TomDufall) [inline README.md:87]:
```suggestion
  <li>You might be missing some cpp libs (software libraries used by the mod); please install 
```

File: README.md [hunk 4]
```diff
@@ -70,6 +78,22 @@ Yes. Every player in the game lobby has to install it.</p>
 I would appreciate any donations. This will help me to develop more mods.
 
 For exclusive modding requests contact me: <a href="mailto:Woodi-dev@gmx.de">Woodi-dev@gmx.de</a>
+
+<h2 id="troubleshooting">Troubleshooting</h2>
+
+<p><b>I can't see <em>loaded</em> message on my game screen</b></br>
+<ol>
+  <li>Make sure you followed all <a href="#installation">instalation steps</a>, especially launching game via Among Us.exe file</li>
```
Comment (TomDufall) [inline README.md:86]:
```suggestion
  <li>Make sure you have followed all the <a href="#installation">installation steps</a>, especially launching the game via the Among Us.exe file</li>
```

File: README.md [hunk 5]
```diff
@@ -70,6 +78,22 @@ Yes. Every player in the game lobby has to install it.</p>
...
+  <li>You migh missing some cpp libs, please install 
+    <a href="https://aka.ms/vs/16/release/vc_redist.x86.exe">visual studio c++</a>
+  </li>
+</ol>
+</p>
+
+<p><b>I can't find my issue.</b></br>
+You can add a <a href="https://github.com/Woodi-dev/Among-Us-Sheriff-Mod/issues/new">github issue</a> explaining your case once loged in.
```
Comment (TomDufall) [inline README.md:94]:
```suggestion
You can <a href="https://github.com/Woodi-dev/Among-Us-Sheriff-Mod/issues/new">raise an issue within GitHub</a> documenting your issue. You will need to be logged into GitHub to do this.
```

**Review by TomDufall — CHANGES_REQUESTED** [review]
A few suggestions for readability/fixing some typos.

**Comment by LeoMouyna** [comment]
> A few suggestions for readability/fixing some typos.

@TomDufall, thanks for your review :)

**Comment by dhari2** [comment]
ع

**Pull request merged**